lib/schlepper/process.rb

```diff
@@ -65,61 +75,72 @@ def say(*); end
         map { |r| [r.fetch('version').to_s, true] }.to_h
     end
 
-    private def load_tasks_that_need_run
+    private def load_tasks_that_need_run only_version: nil
       Dir.glob("#{Rails.root}/#{Paths::TASK_DIR}/*.rb").
         map { |f| File.basename(f) }.
         sort.
-        reject { |f| f.scan(/\A(\d{10,})/).empty? }.
-        reject { |f| @versions.has_key?(f.scan(/\A(\d{10,})/).first.first) }.
-        each { |f| require File.join(Rails.root, 'script', 'schleppers', f) }
-    end
+        each { |f|
+          next if f.scan(/\A(\d{10,})/).empty? ||
+            @versions.has_key?(f.scan(/\A(\d{10,})/).first.first)
 
-    private def process_one klass
-      task = klass.new
+          if only_version
+            next if f.include? only_version
+          end
+
+          require File.join(Rails.root, 'script', 'schleppers', f)
+        }
+   end
+
```

Comment:
The indentation looks off

Reply:
done. haven't found an editor that handles `private def bla` yet.